src/app/@skolera/services/rules.services.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { baseUrl } from 'src/environments/environment';
+
+
+@Injectable()
+export class RulesSerivce {
+    constructor(
+        private http: HttpClient) {
+
+    }
+    getRules(params:any) {
+        return this.http.get(baseUrl + `api//hr/rules`,{ params: params });
```

Comment:
Please remove the extra `/` and for all the other requests

Reply:
done